PWGGA/GammaConv/AliAnalysisTaskNeutralMesonToPiPlPiMiNeutralMeson.cxx

```diff
@@ -822,6 +840,10 @@ void AliAnalysisTaskNeutralMesonToPiPlPiMiNeutralMeson::UserCreateOutputObjects(
   fHistoGammaGammaInvMassPtBeforeCuts     = new TH2F*[fnCuts];
   fHistoMotherInvMassPt                   = new TH2F*[fnCuts];
   fHistoMotherInvMassPtRejectedKinematic  = new TH2F*[fnCuts];
+  fHistoDalitzPlotPosFixedPzNDM          = new TH2F*[fnCuts];
```

Comment:
Perhaps these histograms should be initialzed and filled only for "fDoMesonQA>0".